packages/components/src/docs/Migration/ui-headline.mdx

```diff
@@ -0,0 +1,49 @@
+# Migration Guide: From `ui-headline` to `sd-headline`
+
+The new `sd-headline` is designed to replace the `ui-headline`. Rather than offering a complex component structure, they primarily focus on providing CSS styles for heading elements.
```

Comment:
```suggestion
The new `sd-headline` is designed to replace the `ui-headline`. Rather than offering a complex component structure, they primarily focus on providing CSS styles for heading elements. Make sure to provide correct semantics yourself, e. g. `<h2>`.
```